models/agent_model.go

```diff
@@ -80,10 +95,11 @@ type Agent struct {
 	ListenPort *uint16 `reform:"listen_port"`
 	Version    *string `reform:"version"`
 
-	Username      *string `reform:"username"`
-	Password      *string `reform:"password"`
-	TLS           bool    `reform:"tls"`
-	TLSSkipVerify bool    `reform:"tls_skip_verify"`
+	Username        *string         `reform:"username"`
+	Password        *string         `reform:"password"`
+	TLS             bool            `reform:"tls"`
+	TLSSkipVerify   bool            `reform:"tls_skip_verify"`
+	ServicesTLSKeys ServicesTLSKeys `reform:"services_tls_keys"`
```

Comment:
Did you check if reform support structs? There is high possibility that it won't work and we should keep this field as `[]byte` or `string`